app/src/main/java/com/cornellappdev/volume/ui/screens/HomeScreen.kt

```diff
@@ -128,9 +128,10 @@ fun HomeScreen(
                         val lazyListState = rememberLazyListState()
 
                         Box(modifier = Modifier.padding(top = 10.dp)) {
+                            val followingArticlesSize = followingArticlesState.articles.size;
                             LazyColumn(
                                 modifier = Modifier
-                                    .height(480.dp)
+                                    .height((120 * followingArticlesSize).dp)
```

Comment:
I think there's a little lag so there may be a better way of doing this that's also more generalized then using the size of the article directly. Since we don't actually need this to be a LazyCoulmn, we can opt for just using a regular column:

`
Column(modifier = Modifier.wrapContentHeight()
                                    .padding(end = 12.dp),
                                verticalArrangement = Arrangement.spacedBy(20.dp),
                            ) {
                                (followingArticlesState.articles).forEach { article ->
                                    CreateHorizontalArticleRow(article) {
                                        onArticleClick(
                                            article,
                                            NavigationSource.FOLLOWING_ARTICLES
                                        )
                                    }
                                }
                            }
`

Reply:
You can also get rid of the AnimatedVisibility stuff that was right below this LazyColumn